enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/metrics/jmx/api/ExtAuthMetricsMXBean.java

```diff
@@ -0,0 +1,57 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.wso2.choreo.connect.enforcer.metrics.jmx.api;
+
+/**
+ * MBean API for ExtAuth Service metrics.
+ */
+public interface ExtAuthMetricsMXBean {
+
+    /**
+     * Getter for total request count.
+     * 
+     * @return long
+     */
+    public long getTotalRequestCount();
```

Comment:
This is about total served request count right? Can we get inflight request count as well? Is it doable?

Reply:
I tried `executor.getTaskCount() - executor.getCompletedTaskCount()` in `EnforcerWorkerPool` to count it. But it count 2 or 3 tasks per request and not reliable. will check for other options.

Reply:
we can keep another counter in `ExtAuthMetrics` and increment it at the start of `ExtAuthService.check` method and decrement in `recordMetric` again. but it will add more overhead for the incoming requests.

Reply:
Added a request counter to reset automatically after every 5 mins